### PR TITLE
Update reverse-proxy.md

### PR DESCRIPTION
**Pull request by ArmanTheParman**
exclude mac for network=host

### CODE REVIEW

File: reverse-proxy.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In order to run Nextcloud behind a web server or reverse proxy (like Apache, Ngi
...
 
-    Another option and actually the recommended way in this case is to use `--network host` option (or `network_mode: host` for docker-compose) as setting for the reverse proxy container to connect it to the host network. If you are using a firewall on the server, you need to open ports 80 and 443 for the reverse proxy manually. By doing so, the default sample configurations that point at `localhost:$APACHE_PORT` should work without having to modify them.
+    In this case, another option, and actually the recommended way, is to use `--network host` option (or `network_mode: host` for docker-compose) as setting for the reverse proxy container to connect it to the host network. Do note that this method is not available on MacOS. If you are using a firewall on the server, you need to open ports 80 and 443 for the reverse proxy manually. By doing so, the default sample configurations that point at `localhost:$APACHE_PORT` should work without having to modify them.
```
Comment (szaimen) [inline reverse-proxy.md:42]: maybe we should say that this is not supported on docker desktop in general?

**Review by szaimen — COMMENTED** [review]
Thanks, just one comment

File: reverse-proxy.md (hunk 1 above)
Comment (ArmanTheParman) [reply reverse-proxy.md:42]: your call, but why? it's only the MacOS version that doesn't support --nework=host.

very annoying, btw, has caused me much headache making my own software compatible for both Mac and Linux

Comment (szaimen) [reply reverse-proxy.md:42]: as far as I know it also does not work on windows and in general on all platforms that use docker deesktop...

Comment (Zoey2936) [reply reverse-proxy.md:42]: maybe see https://docs.docker.com/network/drivers/host/#docker-desktop

**Comment by ArmanTheParman** [comment]
OK, I learned something new

**Comment by szaimen** [comment]
Can be closed iirc

**Pull request closed without merging**